reframe/frontend/printer.py

```diff
@@ -61,6 +61,19 @@ def __init__(self):
     def __repr__(self):
         return debug.repr(self)
 
+    def set_verbose_level(self, number):
+        if number:
+            for hdlr in self._logger.stream_handlers():
```

Comment:
Using whatever Python's logging provides us, here you would go over all the handlers and pick only those that are instance of `logging.StreamHandler` and their `stream` is `sys.stdout` (should we really bother for `sys.stderr` here?) and set their level.